godb/molecule/default/molecule.yml

```diff
@@ -5,13 +5,9 @@ driver:
   name: docker
 platforms:
   - name: instance
-    image: docker.io/mpaivabarbosa/molecule-systemd-ubuntu
-    volumes:
-      - /sys/fs/cgroup:/sys/fs/cgroup:ro
-      - /var/lib/docker
+    image: ubuntu:20.04
```

Comment:
Have you tried https://github.com/lean-delivery/docker-systemd ? It's worked for me in the past.

Reply:
Yes I have used that in the past but had lots of issues with Docker and M1 support, so switched to the main ubuntu images :(

Reply:
Let's leave it for now then, but we should try and fix it to avoid our CI drifting too far from production